src/client/components/pages/utils/getHostname.ts

```diff
@@ -0,0 +1,7 @@
+// Sinon does not allow stubbing of the window.location, so it is necessary to
+// access this value from here so that we can stub the window location in a test
```

Comment:
This is another minor point, but the docblock format (starting with two asterisks, `/**`) is necessary for this to get picked up by our documentation generator, while the inline `//` comments get ignored.

```suggestion
/**
 * Sinon does not allow stubbing of the window.location, so it is necessary to
 * access this value from here so that we can stub the window location in a test
 */
```

Reply:
Ah good to know. Updated this in [45034bc](https://github.com/seas-computing/course-planner/pull/529/commits/45034bcf532f4b90094c613a3ac6468d9ec90123).